contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,925 @@
+---
+date: 2022-12-05
+title: PostHog vs Mixpanel
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Guides", "Comparisons"]
+---
+
```

Comment:
I think we should consider adding a CTA here which links to other comparisons. Or could go on the end.